app/pb_kits/playbook/pb_avatar/docs/_avatar_monogram.md

```diff
@@ -0,0 +1 @@
+## Monograms are useful too!
```

Comment:
It would be good to launch with real content. Why not get some from @christinaatai